package/MDAnalysis/analysis/base.py

```diff
@@ -90,13 +90,76 @@
    plt.ylabel("RMSD (Å)")
 
 
+Using parallelization in your analysis
+-------------------------------------
+--------------------------------------
+Starting from 2.8.0, MDAnalysis ``AnalysisBase`` subclasses can run on a backend
+that supports parallelization (see :mod:`MDAnalysis.analysis.backends`). By
+default, only one backend is supported -- built-in `multiprocessing`, that
+processes parts of a trajectory running separate *processes*, i.e. utilizing
+multi-core processors properly. For now, parallelization is added to
```

Comment:
would be good to clarify that by default only the serial backend is enabled, which is the same as the current behaviour.